charts/argocd-apps/ci/application-templates.yaml

```diff
@@ -0,0 +1,38 @@
+applicationTemplates:
+  - apps:
+      - name: guestbook
+        project: guestbook-project
+    template: |-
```

Comment:
Does it also work with YAML/object rather than type string?

Reply:
yes, it does